src/Components/Sort.jsx

```diff
@@ -0,0 +1,24 @@
+import { getArticles } from "../Utils/Api"
+
+
+export const Sort = ({articles, setArticles}) => { 
+
+
+const sortBy = (event) => {
+    event.preventDefault() 
+    getArticles(event.target.value).then(({data})=>{
+        console.log(data.articles)
+        setArticles(data.articles)
+    })
+}
+
+
+    return (<section>
+        <select className="sort-by" onChange={(sortBy)}>
+            <option>Sort by...</option>
```

Comment:
this option returns an API error when selected after other sort functions; consider just having a default sort option.